h2o-core/src/main/java/water/api/schemas3/ParseV3.java

```diff
@@ -69,4 +70,7 @@
   // Zero if blocking==false; row-count if blocking==true
   @API(help="Rows", direction=API.Direction.OUTPUT)
   public long rows;
+
+  @API(help="One-character string used to escape other characters.", direction=API.Direction.INOUT)
+  public byte escapechar = ParseSetup.DEFAULT_ESCAPE_CHAR;
```

Comment:
`byte` at API level? I'm amazed that it works.
In that case, maybe help should be updated to "One ASCII character used to..."

Reply:
we have also `public byte separator`, so did it in the same way / same validation